### DISCUSSION TITLE
Suddenly lost access to student developer pack, while academic verification is still valid

### DISCUSSION THREAD

**Discussion by JamesJieranShen** [community/community#88416] [GitHub Education]
### Hi everyone,

Greetings! I am a university student and have been using the student developer pack for a while now. However, I have suddenly lost access to my student developer pack perks, including github pro and copilot. When I log on to education.github.com, I am seeing that I have been a verified student since Jan. 2023.  I have tried to re-verify as well, but the webpage claims that there's no need to verify.  However, under student developer pack offers, it now says "Your Student Developer Pack will be ready soon!". I'm not sure what has happened to cause this problem.

**Comment by Ashishdheer** [comment]
I think that you're experiencing a disruption in your GitHub Student Developer Pack perks despite being verified as a student. For this issue, start by reaching out to GitHub Support through the support page after logging in with your GitHub account. Clearly explain the problem, including details about your student verification and the status of your perks. Check your GitHub account email for any relevant messages and review GitHub Education documentation for updates or troubleshooting steps. Be patient, as these issues may take time to resolve, and consider following up with GitHub Support if the problem persists. If available, you can also seek assistance from your university's IT or technical support team.
